Applications/CMC/cmc.cpp

```diff
@@ -71,6 +72,11 @@ int main(int argc,char **argv)
             return(0);
  
         // PRINT A DEFAULT SETUP FILE FOR THIS INVESTIGATION
+        } else if(startsWith("-j",option.c_str())){
+            char* optionCStr = const_cast<char*>(option.c_str());
+            optionCStr += 2; //increment the char*'s memory address by 2, skip "-j"
+            specifiedMaxNumThreads = atoi(optionCStr);
+
```

Comment:
If we do so I'd like for us to use a command line argument parsing library, perhaps from boost or http://tclap.sourceforge.net/; I've played with TCLAP for the predictive sims and it works really well.

I also want to consolidate our executables into one executable:

```
opensim cmc -s ...
opensim ik -s ...
```

like one does with git. 

Sorry for hijacking your PR, Thomas.